### PR TITLE
Original code in Auth branch

### PR DESCRIPTION
**Pull request by theref**
This is the original code from the auth branch written by @KPrasch @theref and @cygnusv

### CODE REVIEW

File: contracts/contracts/coordination/Coordinator.sol [hunk 1]
```diff
@@ -252,9 +251,6 @@ contract Coordinator is AccessControlDefaultAdminRules {
         return id;
     }
 
-<<<<<<< HEAD
-    function cohortFingerprint(address[] calldata nodes) public pure returns (bytes32) {
-=======
```
Comment (KPrasch) [inline contracts/contracts/coordination/Coordinator.sol]: Looks like a merge conflict made it's way into the diff here.

File: tests/test_coordinator.py [hunk 2]
```diff
@@ -15,11 +15,12 @@
     "RitualState",
     [
         "NON_INITIATED",
+        "INITIATED",
         "AWAITING_TRANSCRIPTS",
         "AWAITING_AGGREGATIONS",
+        "FINALIZED",
         "TIMEOUT",
         "INVALID",
-        "FINALIZED",
```
Comment (KPrasch) [inline tests/test_coordinator.py]: Why the reordering here?

File: contracts/contracts/coordination/AllowList.sol [hunk 3]
```diff
@@ -31,23 +39,27 @@ contract AllowList is AccessControlDefaultAdminRules, IRitualAuthorizer, RitualS
...
             "Only active rituals can add authorizations");
         for (uint i=0; i<addresses.length; i++) {
             authorizations[ritualId][addresses[i]] = true;
         }
+        feeRecipient.transfer(msg.value);
     }
 
-    function deauthorize(uint32 ritualId, address[] calldata addresses) public {
+    function deauthorize(uint32 ritualId, address[] calldata addresses) public payable {
+        require(msg.value > 10000, "Insufficient fee");
```
Comment (KPrasch) [inline contracts/contracts/coordination/AllowList.sol]: Hmm, why send ether to this function?

File: tests/test_coordinator.py (hunk 2 above)
Comment (theref) [reply tests/test_coordinator.py]: because it needs to match the contract

File: contracts/contracts/coordination/AllowList.sol (hunk 3 above)
Comment (theref) [reply contracts/contracts/coordination/AllowList.sol]: This is an example of a payment gateway. If an adopter wants to edit the authorization they need to pay

File: contracts/contracts/coordination/Coordinator.sol (hunk 1 above)
Comment (theref) [reply contracts/contracts/coordination/Coordinator.sol]: seems outdated? i can't find that

**Pull request closed without merging**